docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,120 @@
+# Release 353 (XXXXXXXX March 2021)
+
+## General
+
+* Add {doc}`/connector/clickhouse`. ({issue}`4500`)
+* Extend support for correlated subqueries including `UNNEST`. ({issue}`6326`, {issue}`6925`, {issue}`6951`)
+* Add {func}`to_geojson_geometry` and {func}`from_geojson_geometry` functions. ({issue}`6355`)
+* Add support for values of any integral type (`tinyint`, `smallint`, `integer`, `bigint`, `decimal(p, 0)`)
+  in window frame bound specification. ({issue}`6897`)
+* Improve query planning time for queries containing `IN` predicates with many elements. ({issue}`7015`)
+* Fix potential incorrect results when columns from `WITH` clause are exposed with aliases. ({issue}`6839`)
+* Fix potential incorrect results for queries containing multiple `<` predicates. ({issue}`6896`)
+* Always show `SECURITY` clause in `SHOW CREATE VIEW`. ({issue}`6720`)
+* Fix reporting of column references for aliased tables in `QueryCompletionEvent`. ({issue}`6972`)
+* Fix potential compiler failure when constructing an array with more than 128 elements. ({issue}`7014`)
+* Fail `SHOW COLUMNS` when column metadata cannot be retrieved. ({issue}`6958`)
+* Fix rendering of function references in `EXPLAIN` output. ({issue}`6703`)
+* Fix planning failure when `WITH` clause contains hidden columns. ({issue}`6838`)
+                   
+## Security
+
+* Break Oauth2 authentication flow as soon as login fails. ({issue}`6659`)
```

Comment:
Can we use a different word than "break"? It's not clear what this means

Reply:
@kokosing ?